gsdk/tests/rpc.rs

```diff
@@ -254,7 +262,7 @@ async fn test_runtime_wasm_blob_version() -> Result<()> {
 
 #[tokio::test]
 async fn test_runtime_wasm_blob_version_history() -> Result<()> {
-    let api = Api::new(Some("wss://rpc.vara-network.io:443")).await?;
+    let api = Api::new(Some("wss://archive-rpc.vara-network.io:443")).await?;
```

Comment:
**Suggestion:** Consider using a constant for the URL 'wss://archive-rpc.vara-network.io:443' to avoid hardcoding.
```suggestion
    const VARA_NETWORK_URL: &str = "wss://archive-rpc.vara-network.io:443";
    ...
    let api = Api::new(Some(VARA_NETWORK_URL)).await?;
```

Reply:
No need for the one-shot test.